src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.cpp

```diff
@@ -64,7 +64,7 @@ void CenterOfBrightness::UpdateState(uint64_t CurrentSimNanos)
     imageBuffer = this->imageInMsg.zeroMsgPayload;
```

Comment:
:small_red_triangle_down: I'd squash this commit in with the previous two as well. They all go together